service-base/src/main/java/org/eclipse/hono/service/http/AbstractHttpEndpoint.java

```diff
@@ -225,4 +235,15 @@ protected final String getDeviceIdParam(final RoutingContext ctx) {
         return ctx.request().getParam(PARAM_DEVICE_ID);
     }
 
+    /**
+     * Get a predicate that checks if a given HTTP status is set to the passed value.
+     *
+     * @param statusToCheckFor The HTTP status, e.g. {@link HttpURLConnection#HTTP_OK}.
+     * @return The predicate.
+     */
+    protected final Predicate<Integer> getHttpStatusPredicate(final int statusToCheckFor) {
```

Comment:
this obviously works but FMPOV it would improve readability a lot if we simply would put the lamba in the place where it is used instead of encapsulating this within a method. In the end, that's one of the reasons why lambdas have been introduced.
Compare
```
doTenantHttpRequest(ctx, tenantId, StandardAction.ACTION_ADD, 
   getHttpStatusPredicate(HTTP_CREATED),
   getLocationHeaderHandler(location));
```
with
```
doTenantHttpRequest(ctx, tenantId, StandardAction.ACTION_ADD, 
   status -> status == HTTP_CREATED,
   getLocationHeaderHandler(location));
```
How many bytes have we saved? Which version is easier to understand?
The same holds true for the *getLocationHeaderHandler()* method FMPOV.

Reply:
Well, we fully agree on that - this is just like I wrote it before. Sometimes PR comments are not so clear...

The only difference with explicit getters for lambdas (or Handlers) is that they get a Javadoc and a (hopefully) good name, which sometimes can be helpful IMHO. In this case it is simple enough that this is not necessary.

Anyway, I will introduce the lambdas again and hope we both are satisfied then...